tools/common_lib/src/layers_utils.h

```diff
@@ -36,15 +36,18 @@ enum class DataType
 {
     eFp32 = 0,
     eFp16 = 1,
+    eUint4 = 2,
     eCount
 };
 
-inline std::uint8_t get_data_type_bytes_width(DataType dt)
+//inline std::uint8_t get_data_type_bytes_width(DataType dt)
+inline float get_data_type_bytes_width(DataType dt)
 {
     switch (dt)
     {
     case DataType::eFp32: return sizeof(float);
     case DataType::eFp16: return sizeof(std::uint16_t);
+    case DataType::eUint4: return 0.5;
```

Comment:
Should be "0.5f", because without "f" literal this indirect cast from double to float

Reply:
also please remove commented line above

Reply:
Done